downstream/assemblies/platform/assembly-aap-platform-components.adoc

```diff
@@ -10,6 +10,16 @@ ifdef::context[:parent-context: {context}]
 [role="_abstract"]
 {PlatformNameShort} is a modular platform composed of separate components that can be connected together to meet your deployment needs. {PlatformNameShort} deployments start with {ControllerName} which is the enterprise framework for controlling, securing, and managing Ansible automation with a user interface (UI) and RESTful application programming interface (API). Then, you can add to your deployment any combination of the following automation platform components:
```

Comment:
This statement isn't correct. It may have been before, but now the listed items below are required.

Can we reconcile the data in the overview of tested deployment models with the list here.

Reply:
Hey @thaumos! I have updated this section accordingly. Are there any other changes needed?